libs/libbat/include/verge/mission-command/BatteryModel.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include <string>
+#include <verge/mission-command/BatteryCoefficients.h>
+
+namespace Verge
+{
+    namespace MissionCommand
+    {
+        class BatteryModel
+        {
+        public:
+            BatteryModel(BatteryCoefficients coefficients);
+            void update(float dt);
+            float getCapacity();
```

Comment:
change this to `float getCapacity() const;`

This tells the user and compiler that nothing changes when you call this method